config/routes.rb

```diff
@@ -16,12 +16,13 @@
   get 'projects', to: 'static_pages#projects'
 
   resources :articles, only: [:index, :show] do
-    # resources :comments
+    resources :comments, only: [:show, :create]
```

Comment:
Do you need a show action?  I would say you need :new and :create

Reply:
Hi Kris, is :create & :new not the same thing?

Reply:
No, new and create are 2 different actions

Reply:
New instantiates the object, create saves it